Packages/io.chainsafe.web3-unity.loader/Editor/ScopedRegistryInstaller.cs

```diff
@@ -0,0 +1,125 @@
+#if UNITY_EDITOR
```

Comment:
this is redundant since it's already in the editor folder, a better way to do it is to have an assembly definition like other packages in here and set platform to only editor